packages/forma-36-react-components/src/components/Switch/style.css

```diff
@@ -0,0 +1,56 @@
+@import 'settings/colors';
+
+.Switch__wrapper {
+  display: flex;
+}
+
+.Switch {
+  width: 35px;
+  height: 21px;
+  background: var(--color-element-mid);
+  border-radius: 21px;
+  cursor: pointer;
+  position: relative;
+  outline: none;
+  margin: 0 0.5rem 0 0;
+  border: 0;
+  appearance: none;
+}
+
+.Switch:before {
+  position: absolute;
+  content: '';
+  width: 17px;
+  height: 17px;
+  left: 2px;
```

Comment:
Would recommend adding a border to the main `.Switch` element so we don't have to play around with positioning like this. https://codepen.io/anon/pen/NmgEMX